### PR TITLE
Enable receipt migration from rpc

### PR DESCRIPTION
**Pull request by LukaszRozmej**

### CODE REVIEW

File: src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugModule.cs [hunk 1]
```diff
@@ -118,6 +119,11 @@ public ResultWrapper<GethLikeTxTrace> debug_traceTransactionInBlockByIndex(byte[
             return ResultWrapper<GethLikeTxTrace>.Success(transactionTrace);            
         }
 
+        public async Task<ResultWrapper<bool>> debug_setReceiptMigrationPoint(long blockNumber)
+        {
+            return ResultWrapper<bool>.Success(await _debugBridge.SetReceiptMigrationPoint(blockNumber + 1));
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugModule.cs:124]: this blockNumber, blockNumber + 1 may lead to confusion?

File: src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugBridge.cs [hunk 2]
```diff
@@ -77,6 +80,11 @@ public int DeleteChainSlice(long startNumber)
             return _blockTree.DeleteChainSlice(startNumber);
         }
 
+        public Task<bool> SetReceiptMigrationPoint(long blockNumber)
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugBridge.cs:83]: rename to run migration

File: src/Nethermind/Nethermind.Cli/Modules/DebugCliModule.cs [hunk 3]
```diff
@@ -98,9 +98,15 @@ public string GetBlockRlp(long number)
         {
             return NodeManager.Post<string>("debug_getBlockRlp", number).Result;
         }
+        
+        [CliFunction("debug", "setReceiptMigrationPoint")]
+        public bool SetReceiptMigrationPoint(long number)
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.Cli/Modules/DebugCliModule.cs:103]: call it migrateReceipts or runReceiptsMigration

Comment (tkstanczak) [reply src/Nethermind/Nethermind.Cli/Modules/DebugCliModule.cs:103]: can we add startBlock, endBlock?

File: src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugModule.cs (hunk 1 above)
Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugModule.cs:124]: adding comment (inclusive->exclusive conversion)

File: src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugBridge.cs (hunk 2 above)
Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugBridge.cs:83]: Not sure about that. It does what it does, sets migration point, and then migrations runs or not depending if it needs to run.

File: src/Nethermind/Nethermind.Cli/Modules/DebugCliModule.cs (hunk 3 above)
Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.Cli/Modules/DebugCliModule.cs:103]: we could in theory but its not worth it, historically we want to migrate all receipts before some point and I don't see any other use case for it

**Pull request merged**